app/src/main/java/io/gnosis/safe/ui/settings/app/passcode/EnterPasscodeFragment.kt

```diff
@@ -52,7 +52,7 @@ class EnterPasscodeFragment : BaseViewBindingFragment<FragmentPasscodeBinding>()
             when (val viewAction = it.viewAction) {
                 is PasscodeViewModel.AllOwnersRemoved -> {
                     snackbar(requireView(), R.string.passcode_disabled)
-                    findNavController().navigateUp()
+                    findNavController().popBackStack(R.id.transactionDetailsFragment, false)
```

Comment:
what if you reset passcode from the settings?

Reply:
When selecting `forgot passcode` in `change passcode` or `disable passcode` I navigate back to the `passcode setting`.